docs/settings/get-settings.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: "Get Settings"
+description: "Gets the settings for the API customer."
+api: "GET /settings"
+---
+
+## Response
+
+<ResponseField name="id" type="string" required>
+  The settings ID.
+</ResponseField>
+
+<ResponseField name="webhookUrl" type="string" required>
+  The customer's webhook URL.
+</ResponseField>
+
+<ResponseField name="webhookKey" type="string" required>
+  The key used to authenticate webhook calls to the customer's app.
+</ResponseField>
```

Comment:
General comment throughout the docs - these docs are for existing customers/devs.

The tone should be "your app" rather than "the customer's app"